src/chat/components/footer/default-footer.tsx

```diff
@@ -216,7 +216,13 @@ export default (props: any) => {
       >
         <PlusWrap>
           <SendBtn
-            onPress={loading || props.loading ? null : () => sheet.show()}
+            onPress={
+              loading || props.loading
+                ? null
+                : props.footerOnPressLaunchIcon
+                ? props.footerOnPressLaunchIcon
+                : () => sheet.show()
+            }
```

Comment:
For this `onPress`, can you try make sure we are not using nested tenary operator, I know we have been using it in the past but moving forward let's try refrain from it.
You can create a function like `handleLaunchIconPress` for it in this case